src/components/v2/V2Create/tabs/FundingCycleTab/FundingCycleTabContent.tsx

```diff
@@ -107,6 +107,7 @@ export default function FundingCycleTabContent({
           setFundingDrawerVisible(false)
         }}
         getContainer={false}
+        destroyOnClose
```

Comment:
this is needed to we force to load from Redux every time the drawer is opened.

Reply:
Yep